src/manifolds/VectorBundle.jl

```diff
@@ -472,6 +479,8 @@ function get_vectors(M::VectorBundleFibers, p, B::CachedBasis)
 end
 Base.@propagate_inbounds Base.getindex(x::FVector, i) = getindex(x.data, i)
 
+base_manifold(TpM::TangentSpaceAtPoint) = TpM.fiber.manifold
```

Comment:
```suggestion
base_manifold(VpM::VectorSpaceAtPoint) = VpM.fiber.manifold
```

Reply:
This contradicts line 211 in the same file, where base_manifold of the vectorslaceatpoint is set to the fiber; what is now the right definition?

Reply:
Ah, I see, both do basically the same, just written differently. We keep the one that already existed.

Reply:
OK, cool, so we don't even need this method.